src/components/PrimaryPanes/Outline.css

```diff
@@ -70,3 +70,19 @@
 .outline-list__element:hover {
   background: var(--theme-toolbar-background-hover);
 }
+
+.outline-footer {
+  background: var(--theme-body-background);
+  border-top: 1px solid var(--theme-splitter-color);
+  position: absolute;
+  display: flex;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  opacity: 1;
+  z-index: 1;
+  -moz-user-select: none;
+  user-select: none;
+  height: 24px /* var(--editor-footer-height) */;
```

Comment:
`--editor-footer-height` needs to be globally available; should also look at how the SecondaryPane handles the footer height, and possibly consolidate